sql/src/test/java/org/apache/druid/sql/calcite/BaseCalciteQueryTest.java

```diff
@@ -633,16 +632,16 @@ public QueryLogHook getQueryLogHook()
   @ClassRule
   public static SqlTestFrameworkConfig.ClassRule queryFrameworkClassRule = new SqlTestFrameworkConfig.ClassRule();
 
-  @Rule
+  @Rule(order = 3)
```

Comment:
why is the order number needed for the test rule?

Reply:
I've described the main problem in [this comment](https://github.com/apache/druid/pull/15465#discussion_r1455534365)

this order=3 ensures that this rule must follow the tempfolder rule so that the `beforeNewFrameworkCreateHook` could use the `tempFolder`